### DISCUSSION TITLE
关于第七章《容器资源限制以及 QoS 等级》中物理资源抽象的讨论

### DISCUSSION THREAD

**Discussion by harrymore** [isno/theByteBook#124] [General]
文中提到：

> Kubernetes 将管理的物理资源抽象成指定的类型和对应的计量单位。 这些资源可以分类两类，可压缩资源和不可压缩资源。

> 可压缩资源：例如` CPU、GPU。当此类资源超限时，Pod 中进程使用 CPU 会被限制，但不会被杀掉。

这里的GPU我觉得欠妥，因为我们在申请GPU资源的时候，申请的是显存而不是GPU的算力，因此它的资源类型其实也是内存（具体点是显存），是会产生OOM 问题的。

**Comment by isno** [comment]
感谢指正，这篇文章我需要重新编辑。
